BraveRewardsUI/Settings/Ads/SettingsAdSectionView.swift

```diff
@@ -130,3 +146,35 @@ class SettingsAdSectionView: SettingsSectionView {
     $0.font = SettingsUX.bodyFont
   }
 }
+
+private class DisabledAdsView: UIStackView {
+  
+  override init(frame: CGRect) {
+    super.init(frame: frame)
+    
+    axis = .horizontal
+    spacing = 10
+    alignment = .center
+    layoutMargins = UIEdgeInsets(top: 15, left: 15, bottom: 15, right: 15)
```

Comment:
`UIEdgeInsets(equalInset: 15)`

Reply:
Can't use these, they are in `Client`, and this is in the `BraveRewardsUI` framework